api/src/pcapi/core/educational/commands.py

```diff
@@ -182,18 +182,21 @@ def notify_reimburse_collective_booking(booking_id: int, reason: str, value: Dec
     )
 
 
+@blueprint.cli.command("synchronise_institutions_geolocation")
```

Comment:
Tu peux en profiter pour corriger :
```suggestion
@blueprint.cli.command("synchronize_institutions_geolocation")
```

Reply:
`synchronise` existe il me semble, c'est la version UK

Reply:
Les 2 sont présents dans le code. La version avec `z` est significativement plus présent -> le propose de laisser en l'état.